titiler/cmr/backend.py

```diff
@@ -73,7 +74,7 @@ class CMRBackend(BaseBackend):
 
     input: str = attr.ib("CMR", init=False)
 
-    _backend_name = "CMR"
+    _backend_name: str = attr.ib(default="CMR")
```

Comment:
why do we need this change?

Reply:
I kept getting linter warnings before making this change. It is possible I have things set incorrectly in my environment, but I think the result is the same with either approach but now I don't get the warning!
![image](https://github.com/user-attachments/assets/32749085-9bf8-43d0-9f5b-cc1553f83dab)